src/test/java/org/gridsuite/modification/server/Impacts/TestImpactUtils.java

```diff
@@ -145,10 +145,14 @@ public static void testBranchImpacts(ObjectMapper mapper, SimpleImpactType impac
                                          String breakerId2, String disconnectorId2, String substationId2) throws JsonProcessingException {
         Optional<NetworkModificationResult> networkModificationResult = mapper.readValue(resultAsString, new TypeReference<>() { });
         assertTrue(networkModificationResult.isPresent());
+        List<SimpleElementImpact> impacts = createBranchImpacts(impactType, branchType, branchId, breakerId1, disconnectorId1, substationId1, breakerId2, disconnectorId2, substationId2);
+        if (impactType != SimpleImpactType.DELETION) {
+            impacts.add(createElementImpact(SimpleImpactType.MODIFICATION, branchType, branchId, new TreeSet<>(List.of(substationId1, substationId2)))); // case with newtapChanger
```

Comment:
Move this in `createBranchImpacts` method ?

Reply:
Done